src/actions/heap/heap.ts

```diff
@@ -123,7 +125,7 @@ export class HeapAction extends Hub.Action {
         type: "select",
       },
       {
-        label: "Heap Field (identity or account_id field)",
+        label: "Heap Field Label (identity or account_id field)",
```

Comment:
connie had this ask too (from DMs) can you make this change while youre here 
![Screen Shot 2021-05-07 at 1 01 57 PM](https://user-images.githubusercontent.com/17321973/117502534-7e79be00-af34-11eb-8484-8919893c9688.png)